src/main/java/com/erigitic/commands/BalanceTopCommand.java

```diff
@@ -98,23 +95,27 @@ public CommandResult execute(CommandSource src, CommandContext args) throws Comm
         final Currency fCurrency = currency;
 
         if (totalEconomy.isDatabaseEnabled()) {
-            try {
-                Statement statement = totalEconomy.getSqlManager().dataSource.getConnection().createStatement();
-                String currencyColumn = currency.getName() + "_balance";
-                statement.execute("SELECT * FROM accounts ORDER BY `" + currencyColumn + "` DESC LIMIT 10");
-                ResultSet set = statement.getResultSet();
-
-                while (set.next()) {
-                    BigDecimal amount = set.getBigDecimal(currencyColumn);
-                    UUID uuid = UUID.fromString(set.getString("uid"));
-                    Optional<User> optUser = Sponge.getServiceManager().provideUnchecked(UserStorageService.class).get(uuid);
-                    String username = optUser.map(User::getName).orElse("unknown");
-
-                    accountBalances.add(Text.of(TextColors.GRAY, username, ": ", TextColors.GOLD, currency.format(amount)));
+            String query = "SELECT * FROM balances WHERE `currency_ident` = '?' ORDER BY `balance` DESC LIMIT 10";
+            try (Connection connection = totalEconomy.getTESqlManager().getCommandConnection();
+                 PreparedStatement statement = connection.prepareStatement(query)) {
```

Comment:
I think there is extra whitespace at the beginning of this line. That should be removed to be in line with the rest of the code.

Reply:
Uh, no. Not really.
That's how IntelliJ formats multi-line try-with-resources initializers.
I think the reason is that it is aligned with the first line this way. (Which also means that it is offset by one char due to the brace.